package.json

```diff
@@ -48,64 +48,60 @@
 		"ci:coverage": "codecov -f coverage/coverage-final.json",
 		"ci:test": "jest --config tools/jest/jest.config.json --runInBand --no-cache"
 	},
-	"peerDependencies": {
-		"@angular/common": ">= 4.0.0 < 5.0.0",
-		"@angular/core": ">= 4.0.0 < 5.0.0"
-	},
 	"devDependencies": {
-		"@angular/common": "4.1.x",
-		"@angular/compiler-cli": "4.1.x",
-		"@angular/compiler": "4.1.x",
-		"@angular/core": "4.1.x",
-		"@angular/platform-browser-dynamic": "4.1.x",
-		"@angular/platform-browser": "4.1.x",
-		"@types/jest": "20.0.x",
+		"@angular/common": "^5.0.3",
```

Comment:
I'd usually prefer `5.0.x` instead of `^5.0.3` as it is easier to understand for developers